src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -66,6 +65,12 @@ public bool IsRecording
             }
         }
 
+        /// <summary>
+        /// Gets a collection of key/value pairs that represents information that is passed
+        /// to children of this <see cref="TelemetrySpan"/>.
+        /// </summary>
+        public IEnumerable<KeyValuePair<string, string>> Baggage => this.Activity?.Baggage;
```

Comment:
I'd remove this change from this PR. It needs to be decided what do we call them. CorrelationContext is being renamed to Baggage, and lets do it separately.

Reply:
just removed!